Octane - Replace some references to Computed Properties with Tracked (#805)
ember-learn/guides-source: guides/release/controllers/index.md

#### When should we create a Controller?

* We want to pass down actions or variables to share with a Route’s child components

> * We have a getter that needs the results of the model hook

Reviewer: I think that just saying "getter" here is a tad odd, we **are** computing a value (just not a `Ember.computed`), what do yo uthink about:

```suggestion
* We need to compute a value based on the results of the model hook
```
